src/Microsoft.AspNetCore.SignalR.Client/Connection.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO.Pipelines;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.SignalR.Client
+{
+    public class Connection : IPipelineConnection
```

Comment:
A PipelineConnection is an object that provides a reader for one pipeline and a writer for another pipeline. That's what our Connection object is, so we want to implement the interface to inherit any extension methods, etc. (though, to be fair, I don't think there are any extension methods on here yet :))